pipeline-model-extensions/src/main/java/org/jenkinsci/plugins/pipeline/modeldefinition/withscript/WithScriptScript.java

```diff
@@ -24,12 +24,15 @@
 
 package org.jenkinsci.plugins.pipeline.modeldefinition.withscript;
 
+import org.jenkinsci.plugins.scriptsecurity.sandbox.whitelists.Whitelisted;
 import org.jenkinsci.plugins.workflow.cps.CpsScript;
 
 import java.io.Serializable;
 
 public abstract class WithScriptScript<T extends WithScriptDescribable<T>> implements Serializable {
+    @Whitelisted
```

Comment:
i.e., it's a flaw that was only uncovered after the main fix here and it made sense to just fold it into the same PR.